### PR TITLE
Add error message if mainWindowUrl was wrong

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexanderMoskovkin @kirovboris @VasilyStrelyaev

### CODE REVIEW

File: src/errors.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import dedent from 'dedent';
+import { render as renderTemplate } from 'mustache';
+import CONSTANTS from './constants';
+
+export default {
+    mainUrlWasNotLoaded: dedent `
+        The page specified by the mainWindowUrl option was not loaded. 
+        It may be the one from the following list:
```
Comment (VasilyStrelyaev) [inline src/errors.js]:
```
        A page in the main window specified by the mainWindowUrl option was not loaded. 
        The affected page is one of the following:
```

File: src/errors.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import dedent from 'dedent';
+import { render as renderTemplate } from 'mustache';
+import CONSTANTS from './constants';
+
+export default {
+    mainUrlWasNotLoaded: dedent `
+        A page in the main window specified by the mainWindowUrl option was not loaded. 
+        The affected page is one of the following:
```
Comment (VasilyStrelyaev) [inline src/errors.js]:
```
        The main window page at ${url} was not loaded. 
        Try specifying one of the following pages that were successfully loaded as the main window page:
```

Comment (VasilyStrelyaev) [inline src/errors.js]: How do you like this one?

Comment (AlexanderMoskovkin) [inline src/errors.js]: It's better but I think we should say where you should specify it (via the mainWindowUrl option)

Comment (VasilyStrelyaev) [inline src/errors.js]:
```
        The main window page at ${url} was not loaded. 
        Use the mainWindowUrl option to specify one of the following pages as the main window page:
```

**Pull request merged**